apps/api/e2e/utils/test-helpers.ts

```diff
@@ -81,20 +138,22 @@ export async function registerUserAndAgentAndGetClient({
   userEmail?: string;
   userImageUrl?: string;
   agentName?: string;
+  agentHandle?: string;
   agentDescription?: string;
   agentImageUrl?: string;
   agentMetadata?: Record<string, unknown>;
   agentWalletAddress?: string;
 }) {
-  const sdk = getApiSdk(adminApiKey);
+  const sdk = new ApiClient(adminApiKey);
```

Comment:
note: i removed the Speakeasy SDK usage here to avoid regenerating it...idk what we want to do there, anymor